services/api/src/modules/payments/payments.service.ts

```diff
@@ -315,6 +459,25 @@ export default class PaymentsService {
     return newPayment
   }
 
+  async getWirePayment(sourceId: string) {
+    // Last 72 hours
+    const newDate72HoursInpast = new Date(
+      new Date().setDate(new Date().getDate() + 3)
+    )
+    // Get recent payments of wire type
+    const payments = await this.circle.getPayments({
+      from: newDate72HoursInpast.toString(),
+      type: CirclePaymentQueryType.wire,
+    })
+    if (!payments) return null
+    // Find payment with matching source ID
+    const sourcePayments = payments.map(
+      (payment) => payment.source.id === sourceId
+    )
+    userInvariant(sourcePayments, 'payments not found', 404)
+    return sourcePayments
+  }
+
```

Comment:
For checking for the presence of a matching payment in Circle - will be used in the upcoming scheduler task